server/events/policy_check_output_generator_test.go

```diff
@@ -0,0 +1,166 @@
+package events_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/logging"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPolicyCheckOutputGenerator(t *testing.T) {
+	t.Run("empty when build plan command fails", func(t *testing.T) {
```

Comment:
nit: add test for when checks are disabled

Reply:
like test it returns an empty PolicyCheckOutputStore{} ? I can add it but I don't see any value in it unless we vary the prCreationTimestamp and the repo name to ensure we only turn it on when the rule in atlantis-dynamic-config is satisfied. However, I think part is handled by the feature allocator package itself.